.vscode/settings.json

```diff
@@ -17,5 +17,6 @@
   "editor.codeActionsOnSave": {
     "source.fixAll.eslint": true
   },
-  "files.eol": "\n"
+  "files.eol": "\n",
+  "typescript.tsdk": "node_modules\\typescript\\lib"
```

Comment:
Is this needed?

Reply:
Yes, this is supposed to make you use our libraries version instead of your machines.
I was having all the time issues, where certain features were marked red. Realized, it was because of TS version.
Hopefully, this fix helps now.